pyproject.toml

```diff
@@ -1,3 +1,78 @@
+# ===== Project info
+
+[project]
+dynamic = ["version"]
+name = "fastplotlib"
+description = "Next-gen fast plotting library running on WGPU using the pygfx rendering engine "
+readme = "README.md"
+license = { file = "LICENSE" }
+authors = [{ name = "Kushal Kolar" }, { name = "Caitlin Lewis" }]
+keywords = [
+    "visualization",
+    "science",
+    "interactive",
+    "pygfx",
+    "webgpu",
+    "wgpu",
+    "vulkan",
+    "gpu",
+]
+requires-python = ">= 3.10"
+dependencies = ["numpy>=1.23.0", "pygfx~=0.9.0", "wgpu>=0.20.0", "cmap>=0.1.3"]
```

Comment:
I think the failure is because this is one line, if each dep is on its own line then the `sed` command in the workflow that uses `pygfx@main` will remove the pygfx line from the file.

Reply:
Also I just closed #776 so we can put the lower version pin of `pygfx>=0.9.0` right here instead.